ansible/README.md

```diff
@@ -191,6 +191,42 @@ ansible-playbook -i environments/<environment> routemgmt.yml
 - To use the API Gateway, you'll need to run `apigateway.yml` and `routemgmt.yml`.
 - Use `ansible-playbook -i environments/<environment> openwhisk.yml` to avoid wiping the data store. This is useful to start OpenWhisk after restarting your Operating System.
 
+### Deploying Using MongoDB
+
+You can use MongoDB as the database backend, and there is an ansible task to deploy a single node MongoDB server for testing and developing
```

Comment:
it's not clear reading this part of the doc if mongo is for entities, activations, or both.